### DISCUSSION TITLE
Deploy is not doing it's job for me.

### DISCUSSION THREAD

**Discussion by FrAllard** [willswire/unifi-ddns#87] [Q&A]
![image](https://github.com/user-attachments/assets/bbc568d0-fffb-40ad-80bf-5dc55fd77ea7)

Great! But after hours of waiting ... Still nothing on the CF side.

![image](https://github.com/user-attachments/assets/7937091a-4e61-4f59-be73-e362d86a507c)

Nothing in my worker.

I was able to sucessfully deploy this to someone else UDM Pro and CF account, 2 weeks ago. The only thing I can think of here is that I'm using the same github account to clone the project and try to deploy the same repo from my github account to an other completely unrelated Cloudflare account from the of 2 weeks ago.

**Comment by jerome2710** [comment]
A bit cheasy, but did you try closing the browser tab and reopening dash.cloudflare.com? I felt like I was looking at cache and after a hard-refresh I now see the worker instead of the "Get started" page.
